### PR TITLE
[RENCI/ncdot-road-safety-client#91] improve register/login

### PR DESCRIPTION
**Pull request by mbwatson**
this adds styling for the login & register views that is consistent with the rest of the UI.

### CODE REVIEW

**Comment by mbwatson** [comment]
@hyi i forgot that i wrote this stuff a while back, before i was out for a month, and didn't complete my PR for these changes. i believe things should still work fine.

File: server/templates/base.html
Comment (hyi) [inline server/templates/base.html]: The line change above is not needed since hsrc.png is the image name on the server.

File: server/rs_core/templates/registration/login.html
Comment (hyi) [inline server/rs_core/templates/registration/login.html]: We'll need to replace these placeholder text with relevant language before merging.

File: server/templates/base.html
Comment (hyi) [inline server/templates/base.html]: I love the name RoHAnn, but unfortunately, they decided to use the name RHF instead, so we'll have to give up the nice acronym and stick with RHF instead.

**Review by hyi — CHANGES_REQUESTED** [review]
@mbwatson Thanks for doing the styling work. It looks great. I left a couple of comments for your consideration.

File: server/templates/base.html
Comment (mbwatson) [reply server/templates/base.html]: this was all before the name change, and i just removed the wrong name when resolving the conflicts. thanks for catching!

File: server/rs_core/templates/registration/login.html
Comment (mbwatson) [reply server/rs_core/templates/registration/login.html]: agreed. i'll probably just remove it for now then.

File: server/rs_core/templates/registration/signup.html
Comment (hyi) [inline server/rs_core/templates/registration/signup.html]: One more place to remove placeholder text. Everything looks great other than this. I'll approve the PR after you remove this. @mbwatson

File: server/templates/base.html
Comment (hyi) [inline server/templates/base.html]: Need to remove this title change (RHF->RoHAnn) as well.

**Review by hyi — APPROVED** [review]
Looks great!

**Pull request merged**
